lib/swig/hawkes/model/base/model_hawkes_list.i

```diff
@@ -5,6 +5,8 @@
 #include "tick/hawkes/model/base/model_hawkes_list.h"
 %}
 
+%include "model.i"
```

Comment:
Did you left this by mistake or is this necessary now?

Reply:
not yet but will be